src/website/views/components/picker-group.pug

```diff
@@ -10,90 +10,108 @@ h3 Base
 p.-text To render a picker group, wrap a series of pickers in a div and apply the class <code>m-pickerGroup</code>.
 .example.-mb--3
   .-p--3
-    .m-pickerGroup
-      each type in ['Radio1','Radio2','Radio3']
-        input.a-picker(type="radio", name='radios', id=`ia-${type}`)
-        label(for=`ia-${type}`)
-          = type
+    fieldset
+      legend.a-label.-mb--1 Legend
```

Comment:
@janibolkvadze can we use an adjacent sibling combinator for the margin instead of applying a utility class to the legend? would need to specify for both `m-picker` and `m-pickerGroup`

```
.a-label,
label,
legend {
  & + .m-picker {
    margin-top: 0.25rem;
  }
}
```